sdk/communication/azure-communication-sms/README.md

```diff
@@ -54,23 +54,44 @@ endpoint = os.getenv('AZURE_COMMUNICATION_SERVICE_ENDPOINT')
 sms_client = SmsClient(endpoint, DefaultAzureCredential())
 ```
 
-### Sending SMS Messages
+### Send a 1:1 SMS Message
 
-Once the client is initialized, the `.send()` method can be invoked:
+Once the client is initialized, the `send` method can be invoked:
 
 ```Python
 from azure.communication.sms import SendSmsOptions
 
 sms_responses = sms_client.send(
-    from_phone_number=PhoneNumberIdentifier("<leased-phone-number>"),
-    to_phone_numbers=["<to-phone-number-1>", "<to-phone-number-2>", "<to-phone-number-3>"],
+    from_="<leased-phone-number>",
+    to="<to-phone-number-1>",
     message="Hello World via SMS",
     enable_delivery_report=True, # optional property
     tag="custom-tag") # optional property
 ```
 
-- `from_phone_number`: An SMS enabled phone number associated with your communication service.
-- `to_phone_numbers`: The phone numbers you wish to send a message to.
+- `from_`: An SMS enabled phone number associated with your communication service.
+- `to`: The phone number you wish to send a message to.
```

Comment:
to can be a list of phone numbers, right?

Reply:
This was a snippet for sending 1:1 messages, so that's why I didn't use language for sending 1:N messages. I can see how this can be unclear. I'll update this to ```The phone number or list of phone numbers you wish to send a message to``` for both 1:1 or 1:N.